packages/react-scripts/config/webpack.config.js

```diff
@@ -15,6 +15,7 @@ const webpack = require('webpack');
 const resolve = require('resolve');
 const PnpWebpackPlugin = require('pnp-webpack-plugin');
 const HtmlWebpackPlugin = require('html-webpack-plugin');
+const ExtensionReloader = require('webpack-extension-reloader');
```

Comment:
This is the closest equivalent of the [webpack dev server](https://webpack.js.org/configuration/dev-server/) which is no longer part of this fork due to not supporting hot reloading via the chrome API's. 

Unfortunately, I could not find that it supports [HMR](https://github.com/rubenspgcavalcante/webpack-extension-reloader/issues/6). Perhaps this is something that will be supported later.